Phonebook.Frontend/src/app/shared/dialogs/bug-report-consent/bug-report-consent.component.html

```diff
@@ -1,75 +1,40 @@
 <div>
-  <h1
-    mat-dialog-title
+  <h2
     i18n="
-      Bug-report-consentComponent|Title of Bug Report Consent Dialog@@BugReportConsentComponentTitle"
+      Bug-report-consentComponent|Sub title of bug report consent
+      dialog@@BugReportConsentComponentSubTitle"
   >
-    We need your help!
-  </h1>
-  <mat-dialog-content>
-    <h2
-      i18n="
-        Bug-report-consentComponent|Sub title of bug report consent
-        dialog@@BugReportConsentComponentSubTitle"
-    >
-      Sending automatic Bug reports
-    </h2>
-    <ng-container
-      i18n="
-        Bug-report-consentComponent|Text of Bug Report consent dialog@@BugReportConsentComponentText"
-    >
-      <p>
-        If an error occurs while the application is running, an anonymous error report is
-        automatically generated. In exceptional cases - since this is a preview - the following
-        personal data can still be found in one of the reports:
-      </p>
-      <mat-list>
-        <mat-list-item>
-          Username (short and full name)
-        </mat-list-item>
-        <mat-list-item>
-          Bookmarks and Last seen people
-        </mat-list-item>
-        <mat-list-item>
-          The Last 20 Actions performed in the app
-        </mat-list-item>
-      </mat-list>
-      <p>
-        The data from the error report is used exclusively for the purpose of analysis and
-        correction of the error by our core development team. The data will not be passed on to
-        third parties.
-      </p>
-      <p>
-        <strong
-          >TL;DR With your permission, you agree that bug reports that may contain your personal
-          data can be sent to our core development team and be used for error analysis and
-          resolution.
-        </strong>
-      </p>
-    </ng-container>
-  </mat-dialog-content>
-  <div class="spacer"></div>
-  <mat-dialog-actions>
```

Comment:
what about the actions?

Reply:
Further on, I'll add the option to add a custom "actions footer" to the bottom of the Dialog template. The removed actions will go in there

Reply:
#647

Reply:
No, please do it in this PR. This is the degradation of the UI and this PR should be about enhancing it.

Reply:
Sorry, I read this too late. I already integrated the footer